### PR TITLE
Enable constfold variable in optimize

### PR DESCRIPTION
**Pull request by hwangdeyu**
fixes https://github.com/onnx/tensorflow-onnx/issues/1792

Signed-off-by: hwangdeyu <dejack953@outlook.com>

### CODE REVIEW

**Comment by hwangdeyu** [comment]
Hi @guschmue , 
I found the `constfold` is no used in tf_optimize() after this change https://github.com/onnx/tensorflow-onnx/commit/5c11ecee6f619dfc60fef956a76c31fe90249037. 

And I find some unit test cases is not needed the const fold. Could you help review this change, thanks!

**Comment by guschmue** [comment]
Oh, this is a pretty old change. The rational was that we don't want to ever run with fold_constant=False since it have a pretty hard dependency on constant folding. 
We want to get rid of fold_constant completely but because tf_optimize() was part of the old api we did not want to break it and just ignore fold_constant.

Basically without fold_constant we can hit all kinds of issues - not worth it.

**Comment by hwangdeyu** [comment]
> Oh, this is a pretty old change. The rational was that we don't want to ever run with fold_constant=False since it have a pretty hard dependency on constant folding. We want to get rid of fold_constant completely but because tf_optimize() was part of the old api we did not want to break it and just ignore fold_constant.
> 
> Basically without fold_constant we can hit all kinds of issues - not worth the trouble.
> 
> If this is related to #1792 - the graph that comes out of the unit test is the one we want since the BiasAdd is all 0 (we don't want the back to back Identity - should have been optimized unless both are in the output list) so the question is more why we did not optimize the BiasAdd from the python script - should have taken the same code path.

I get it. We always need `fold_constant` in `tf_optimize()`.
However, the python script runs the different path to ONNX. It deals with the constat folding in  https://github.com/onnx/tensorflow-onnx/blob/72d64606d9aff4f93bb064ba901af49377266150/tf2onnx/tfonnx.py#L468

And calls `fold_constants_using_tf()` instead unit test `rewrite_options.optimizers` to do constfold.
Is it necessary to add the `rewrite_options.optimizers` it in python script path too?

File: tf2onnx/tf_loader.py [hunk 1]
```diff
@@ -684,8 +684,10 @@ def tf_optimize_grappler(input_names, output_names, graph_def, fold_constant=Non
     #   depends on so for now don't turn this on.
     rewrite_options.optimizers[:] = [
         # 'pruning', 'constfold', 'arithmetic', 'dependency', 'function',
-        'constfold', 'function'
+        'function'
     ]
+    if fold_constant:
```
Comment (hwangdeyu) [inline tf2onnx/tf_loader.py:689]: Old version api. `fold_constant` should always be opened.

**Pull request closed without merging**